docs/docs/policies.mdx

```diff
@@ -459,8 +491,9 @@ conversation just before each action was taken.
 
 Every event in a trackers history creates a new state (e.g. running a bot
 action, receiving a user message, setting slots). Featurizing a single state
-of the tracker has a couple steps:
+of the tracker has two steps:
 
+<!-- TODO: below bullet points are not appearing - showing as one block of text instead --->
```

Comment:
it looks to me that this is a theme issue. I think @lunelson has this on his TODO

Reply:
Ah ok super!

Reply:
Lu fixed this in 0.8.0 version of the them. I'll remove the TODO